lib/Jsrt/ChakraCommon.h

```diff
@@ -500,7 +500,12 @@ typedef UINT32 DWORD;
         /// <summary>
         ///     Specified script is internal and non-user code. Hidden from debugger
         /// </summary>
-        JsParseScriptAttributeLibraryCode = 0x1
+        JsParseScriptAttributeLibraryCode = 0x1,
+        /// <summary>
+        ///     ChakraCore assumes ExternalArrayBuffer is Utf8 by default.
+        ///     This one needs to be set for Utf16
+        /// </summary>
+        JsParseScriptAttributeUtf16Encoding = 0x2,
```

Comment:
Being explicit is better,  though that will make this a long name but we should make it JsParseScriptAttributeArrayBufferIsUtf16Encoded

---
In reply to: [85450903](https://github.com/Microsoft/ChakraCore/pull/1830#discussion_r85450903) [](ancestors = 85450903)